packages/babel-core/test/fixtures/parse/output-with-filename.json

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
This file is a copy of `output.json`, but every loc has the `filename` property.